src/pages/detailProduct/index.js

```diff
@@ -33,13 +34,19 @@ export default function Index() {
         }
         getData();
     }, []);
+
+    const handleCurrentImage = (index) => {
+        setCurrentImage(index)
```

Comment:
this is one linem, you can pass directly the update state

Ej: `const handleCurrentImage = (index) => setCurrentImage(index);`